Cargo.toml

```diff
@@ -361,51 +361,63 @@ lto = "fat"
+sp-consensus = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-consensus-slots = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-core = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-crypto-hashing = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-database = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-debug-derive = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-externalities = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-inherents = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-io = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-keystore = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-runtime = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-runtime-interface = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-state-machine = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-std = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-storage = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-timestamp = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-trie = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-version = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+sp-weights = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+staging-xcm = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+substrate-prometheus-endpoint = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+xcm-procedural = { git = "https://github.com/subspace/polkadot-sdk", rev = "2d0d460c37f6816d8bcdea7d6f35eeba72a42c44" }
+
+# Patch away `libp2p-identity` in our dependency tree with the git version.
+# For details see: https://github.com/autonomys/rust-libp2p/blob/04c2e649b1f5482b8c3466b0fdbd1815b3126a48/Cargo.toml#L140-L145
+libp2p-identity = { git = "https://github.com/autonomys/rust-libp2p", rev = "04c2e649b1f5482b8c3466b0fdbd1815b3126a48" }
```

Comment:
I don't think this is the correct section for this. Shouldn't it be under `[patch.crates-io]` only? Sections are for package sources, `libp2p-identity` doesn't come from upstream or downstream `polkadot-sdk` repos.

Reply:
You're right, removing them doesn't actually change anything in the lock file.